### PR TITLE
Add separate CheckpointDataFile

### PR DESCRIPTION
**Pull request by Qwertycoin**
-Removed the CheckpointData from lib/CryptoNoteConfig.h
-Add lib/CryptoNoteCheckpoints.h

**This PR requires a few changed Lines in** 
-https://github.com/qwertycoin-org/qwertycoin-zero
-https://github.com/qwertycoin-org/qwertycoin-gui

**Add a CopyPasta Template in the Readme for new created Files [here](https://github.com/qwertycoin-org/qwertycoin/blob/94baeeed34f42dcd1d2aeb7dc44dbd5d7716fc4d/README.md#copypasta-for-license-when-editing-files)**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,15 +11,16 @@
...
 # Table of contents
 1. [Project Specs](#coinspecs)
-2. [Downloads](#downloads)
-3. [Useful Links](#usefullinks)
-4. [How to Compile Qwertycoin](#howtocompile)
+2. [How to Compile Qwertycoin](#howtocompile)
     1. [Qwertycoin for Linux](#build-linux)
     2. [Qwertycoin for Windows](#build-windows)
-    3. [Qwertycoin for Apple](#build-apple)
+    3. [Qwertycoin for MacOS](#build-apple)
```
Comment (speqtr) [inline README.md]:
```suggestion
    3. [Qwertycoin for macOS](#build-apple)
```

Comment (speqtr) [reply README.md]: Apple officially calls it macOS (starting with lower case **m**). I think we should follow this pattern 🙂

File: README.md [hunk 2]
```diff
@@ -221,6 +192,48 @@ XMR: 47gmN4GMQ17Veur5YEpru7eCQc5A65DaWUThZa9z9bP6jNMYXPKAyjDcAW4RzNYbRChEwnKu1H3
 ETN: etnkJXJFqiH9FCt6Gq2HWHPeY92YFsmvKX7qaysvnV11M796Xmovo2nSu6EUCMnniqRqAhKX9AQp31GbG3M2DiVM3qRDSQ5Vwq
 ```
 
+### Downloads <a name="downloads"></a>
+| Operating System | Download                                 |
+| ---------------- | ---------------------------------------- |
+| Windows 32       | https://releases.qwertycoin.org/get/cli/latest/win32 |
+| Windows 64       | https://releases.qwertycoin.org/get/cli/latest/win64 |
+| Mac 10.8 & Later | https://releases.qwertycoin.org/get/cli/latest/macos |
```
Comment (speqtr) [inline README.md]:
```suggestion
| macOS 10.8 & Later | https://releases.qwertycoin.org/get/cli/latest/macos |
```

File: README.md [hunk 3]
```diff
@@ -221,6 +192,48 @@ XMR: 47gmN4GMQ17Veur5YEpru7eCQc5A65DaWUThZa9z9bP6jNMYXPKAyjDcAW4RzNYbRChEwnKu1H3
...
+| Linux 32         | https://releases.qwertycoin.org/get/cli/latest/linux32 |
+| Linux 64         | https://releases.qwertycoin.org/get/cli/latest/linux64 |
+| OpenBSD 64       | https://releases.qwertycoin.org/get/cli/latest/openbsd64 |
+| FreeBSD 64       | https://releases.qwertycoin.org/get/cli/latest/freebsd64 |
+| Linux ARM 64     | https://releases.qwertycoin.org/get/cli/latest/arm64 |
+| More Builds      | https://releases.qwertycoin.org |
```
Comment (speqtr) [inline README.md]:
```suggestion
| Other platforms      | https://releases.qwertycoin.org |
```

Comment (speqtr) [reply README.md]: Just a suggestion. I think it may be easier to understand for non-technical people 🙂

**Comment by Qwertycoin** [comment]
Thanks @speqtr i will follow your suggestions and changed the Readme.

**Pull request merged**